Upstream: No (Needs a proper solution upstream)
Source: Bernd Steinhauser <berniyh@exherbo.org>
Reason: Preventing startup crashes

From c71ca663fd42522c5d7f15296e42633056c9f827 Mon Sep 17 00:00:00 2001
From: Bernd Steinhauser <berniyh@exherbo.org>
Date: Thu, 18 Jun 2009 00:34:35 +0000
Subject: [PATCH] disable to prevent crashes.

---
 sim/CMakeLists.txt |    4 ++--
 sim/sim.cpp        |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/sim/CMakeLists.txt b/sim/CMakeLists.txt
index db1f827..b563234 100644
--- a/sim/CMakeLists.txt
+++ b/sim/CMakeLists.txt
@@ -7,7 +7,7 @@ ENDIF(NOT WIN32)
 ######################
 
 SET(sim_LIB_SRCS
-	    aboutdata.cpp
+		#aboutdata.cpp
 		ballonmsg.cpp
 		buffer.cpp
 		builtinlogger.cpp
@@ -49,7 +49,7 @@ SET(sim_LIB_SRCS
 		qzip/unzip.c
 	)
 SET(sim_LIB_HDRS
-	    aboutdata.h
+		#aboutdata.h
 		ballonmsg.h
 		buffer.h
 		builtinlogger.h
diff --git a/sim/sim.cpp b/sim/sim.cpp
index 4b9c48e..7718470 100644
--- a/sim/sim.cpp
+++ b/sim/sim.cpp
@@ -307,7 +307,7 @@ int main(int argc, char *argv[])
     }
 #endif
     qInstallMsgHandler(simMessageOutput);
-
+/*
     KAboutData aboutData(PACKAGE,
                          I18N_NOOP("Sim-IM"),
                          _VERSION,
@@ -322,7 +322,7 @@ int main(int argc, char *argv[])
     aboutData.addAuthor("Vladimir Shutoff"		 ,I18N_NOOP("Original Author"),		"vovan@shutoff.ru");
     aboutData.addAuthor("Christian Ehrlicher"	 ,I18N_NOOP("Developer"),			"Ch.Ehrlicher@gmx.de");
     setAboutData(&aboutData);
-
+*/
 #ifndef WIN32
     int _argc = 0;
     char **_argv = new char*[argc + 1];
-- 
1.6.3.2

